lib/mocha.js

```diff
@@ -277,15 +283,67 @@ Mocha.prototype.invert = function() {
  * @param {Boolean} ignore
  * @return {Mocha}
  * @api public
- * @param {boolean} ignore
- * @return {Mocha}
  */
```

Comment:
I agree. 👍 

Since I tend to make atomic commits, I often just cherry-pick them into `master` and delete them from the branch before creating the PR.